.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -0,0 +1,6 @@
+<component name="InspectionProjectProfileManager">
```

Comment:
Switched to [WebStorm](https://www.jetbrains.com/webstorm/), so there are a few config files in here and they'll likely get amended over time as needed.